packages/react/src/components/pickers/TagPicker/TagItem.tsx

```diff
@@ -33,23 +33,31 @@ export const TagItemBase = (props: ITagItemProps) => {
     disabled,
   });
 
+  const itemId = getId();
```

Comment:
Since this is a function component, I believe you'll want to use the `useId` hook to prevent this from being regenerated each time it rerenders.